zipline/pipeline/data/dataset.py

```diff
@@ -115,14 +118,15 @@ class BoundColumn(LoadableTerm):
     inputs = ()
     window_safe = True
 
-    def __new__(cls, dtype, missing_value, dataset, name):
+    def __new__(cls, dtype, missing_value, dataset, name, ndim):
```

Comment:
@llllllllll I tried adding the `ndim` property after our discussion, but `ndim` is being set in `_init` of the `Term` base class, so I wasn't sure how to work around setting an attribute defined as a property